internal/hcs/process.go

```diff
@@ -378,6 +371,10 @@ func (process *Process) CloseStdin() (err error) {
 		return makeProcessError(process, operation, ErrAlreadyClosed, nil)
 	}
 
+	if process.stdin != nil {
```

Comment:
I'm wondering if we should close this after the modify request since it can fail. Seems odd to close the local, then fail in the platform and return the error? Should we ever keep open the platform handle or should we modify close these for all processes and just use our pipe handles?

Reply:
I can move it afterward, sure.